test-e2e/tests/seo_test.ts

```diff
@@ -65,47 +65,76 @@ Scenario('It renders the correct meta tags for the series screen', async ({ I })
   await I.openVideoCard(constants.primitiveAnimalsTitle);
   I.see('Primitive Animals');
 
-  await checkMetaTags(I, 'Blocking', constants.primitiveAnimalsDescription, true);
+  await checkMetaTags(I, 'Primitive Animals', constants.primitiveAnimalsDescription, 'series');
+});
+
+Scenario('It renders the correct meta tags for the episode screen', async ({ I }) => {
+  await I.openVideoCard(constants.primitiveAnimalsTitle);
+  I.see('Primitive Animals');
+
+  I.click('div[aria-label="Play Blocking"]');
+
+  await checkMetaTags(I, 'Blocking', constants.primitiveAnimalsDescription, 'episode');
 });
 
 Scenario('It renders the correct structured metadata for the series screen', async ({ I }) => {
   await I.openVideoCard(constants.primitiveAnimalsTitle);
   I.see('Primitive Animals');
 
-  const url = await I.grabCurrentUrl();
-  const seriesId = getQueryParam(url, 'seriesId');
+  I.seeTextEquals(
+    JSON.stringify({
+      '@type': 'TVSeries',
+      '@id': `${constants.baseUrl}s/${constants.primitiveAnimalsId}`,
```

Comment:
Can you retrieve the expected ID programatically in the test? I don't like having ID's hard coded when we can avoid it.